export const createNewUser = async (req, res, next) => {
  try {
  } catch (err) {
    console.log(err);
  }
};
